docs/docs/en/guide/resource/file-manage.md

```diff
@@ -0,0 +1,68 @@
+# File Management
+
+When third party jars are used in the scheduling process or user defined scripts are required, these can be created from this page. The types of files that can be created include: txt, log, sh, conf, py, java and so on. Files can be edited, renamed, downloaded and deleted.
+
+![file-manage](/img/new_ui/dev/resource/file-manage.png)
+
+- Create a file
+  > The file format supports the following types: txt, log, sh, conf, cfg, py, java, sql, xml, hql, properties.
+
+![create-file](/img/new_ui/dev/resource/create-file.png)
+
+- upload files
+
+> Upload file: Click the "Upload File" button to upload, drag the file to the upload area, the file name will be automatically completed with the uploaded file name.
+
+![upload-file](/img/new_ui/dev/resource/upload-file.png)
+
+- File View
+
+> For the files that can be viewed, click the file name to view the file details.
+
+<p align="center">
+   <img src="/img/file_detail_en.png" width="80%" />
+ </p>
```

Comment:
could you use markdown syntax instead of HTML?

Reply:
Same as other places

Reply:
PTAL @QuakeWang

Reply:
I have updated it.